pkg/webui/components/form/index.js

```diff
@@ -22,7 +22,7 @@ import {
 } from 'formik'
 import scrollIntoView from 'scroll-into-view-if-needed'
 import { defineMessages } from 'react-intl'
-import { isPlainObject, isFunction, pick, omitBy, pull, merge } from 'lodash'
+import { isPlainObject, isFunction, pick, omitBy, pull, merge, isEmpty } from 'lodash'
```

Comment:
I think linting complains that this was added.

Reply:
Indeed